study.md

```diff
@@ -43,7 +43,7 @@ Although both Array and Enumerable have a `count` method, Array also defines the
 method `length`.  Why is `length` sensibly defined on Array but not Enumerable?
 
 ```md
-<!-- your answer here -->
+An enumerable must have an 'each' method, so it goes through the values one at a time.  It can't instantly return the length because it must go through each value individually.
```

Comment:
`length` makes sense on `Array` because arrays have a finite length.  It does not make sense for an `Enumerable` to have a `length` because it might not be finite.  For example, an enumerable that returns a digit of π on each iteration does not have a finite length.